lib/flutter_animation_progress_bar.dart

```diff
@@ -66,7 +66,13 @@ class _FAProgressBarState extends State<FAProgressBar>
   void triggerAnimation() {
     setState(() {
       _currentBegin = _animation.value;
-      _currentEnd = widget.currentValue / widget.maxValue;
+      
+      if (widget.currentValue == 0 && widget.maxValue == 0) {
```

Comment:
Hi @mkranus,

Do you think line 70 should be || (OR) operator? That would make sure for all bad case in math.